sdk/core/azure-core/src/main/java/com/azure/core/util/BinaryData.java

```diff
@@ -187,17 +187,24 @@ public final class BinaryData {
     }
 
     static {
-        BinaryDataHelper.setAccessor(new BinaryDataHelper.BinaryDataAccessor() {
-            @Override
-            public BinaryData createBinaryData(BinaryDataContent content) {
-                return new BinaryData(content);
-            }
+        try {
```

Comment:
At least for the hotfix to debug and in the static ctor below I would rather use System.out.println - simply because any dependency error could be triggered by the definition of the LOGGER field above already.